Services/DataX.Flow/DataX.Flow.Common/EngineEnvironment.cs

```diff
@@ -121,7 +121,10 @@ public async Task<ApiResult> GetEnvironmentVariables()
 
                 FlowBlobConnectionString = KeyVault.GetSecretFromKeyvault(serviceKeyvaultName, flowConfigObj.ConfiggenSecretPrefix + flowConfigObj.StorageAccountName + "-blobconnectionstring");
                 OpsBlobConnectionString = KeyVault.GetSecretFromKeyvault(serviceKeyvaultName, flowConfigObj.ConfiggenSecretPrefix + flowConfigObj.OpsStorageAccountName + "-blobconnectionstring");
-                SparkConnInfo = Helper.ParseConnectionString(Helper.PathResolver(flowConfigObj.SparkConnectionString));
+                if (EngineFlowConfig.SparkType != DataX.Config.ConfigDataModel.Constants.SparkTypeDataBricks)

```

Comment:
For databricks, sparkConnInfo is not required as we dont rely on sparkConnInfo to connect to databricks. For HDInsight this provides the username and password to HDInsight cluster. The reason to add it in the if clause is that Helper.ParseConnectionString throws when the connection string does not contain username and password. This can be left empty in databricks scenario